Updated content for Supported Queries (#2382)
couchbase/docs-server: modules/fts/pages/fts-supported-queries-match.adoc

For example, an operator value of `"and"` means the match query text would be treated like `location` AND `hostel`.  
The default operator value of `"or"` means the match query text would be treated like `location` OR `hostel`.

> A match query is also demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].

Reviewer: ```suggestion
A demonstration of the match query using the Java SDK can be found in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

Author: updated
